BioPage.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Peronal Portfolio Website</title>
+    <link rel="stylesheet" href="styles.css"/>
+</head>
+<body>
+    <div class="main">
+        <nav> 
+            <ul>
+                <li><a href="HomePage.html">HOME</a></li>
+                <li><a href="BioPage.html">ABOUT ME</a></li>
+                <li><a href="ExperiencePage.html">EXPERIENCE PAGE</a></li>
+                <li><a href="PortfolioPage.html">PORTFOLIO PAGE</a></li>
+            </ul>
+        </nav>
+        <div class="homeInfo">
```

Comment:
consider using semantic elements instead of divs (when applicable) in order to make your site more accessible: https://www.w3schools.com/html/html5_semantic_elements.asp